source/documentBase.py

```diff
@@ -140,6 +140,10 @@ def _getNearestTableCell(self, tableID, startPos, origRow, origCol, origRowSpan,
 				destCol+=1 if movement=="next" else -1
 		raise LookupError
 
+	lastTableSelection = None
+	lastTableAxis = None
+	lastTableCoordinate = None

```

Comment:
Have you considered using four variables here, lastRow, lastRowSpan, lastCol and lastCOlSpan? While it adds two extra variables, I think it makes things a bit easier to follow.

Also, please make these private variables, i.e. add add an underscore prefix